modules/compute/function-app/test/main.test.bicep

```diff
@@ -0,0 +1,90 @@
+
+param name string = deployment().name
+param location string = resourceGroup().location
+
+param tags object = {
+  LOB: 'ENT'
+  contact: 'iep.dev@nuance.com'
+  costcenter: '000'
+  environment: 'dev'
+  location: resourceGroup().location
+  team: 'IEP'
+}
+
+
+module dependencies 'dependencies.bicep' = {
+  scope: resourceGroup()
+  name: 'dependencies-${uniqueString(resourceGroup().id)}'
+  params: {
+    name: name
+    location: location
+    tags: tags
+  }
+}
+
+
```

Comment:
```suggestion

module test0 '../main.bicep' = {
  name: 'func-app0-${guid(name)}'
  dependsOn: [
    dependencies
  ]
  params: {
    name: 'funcapp0-${resourceGroup().name}'
    location: location
  }
  scope: resourceGroup()
}
```

Reply:
i like to include a test that provides the most minimum amount of input parameters.